soccer/Processor.cpp

```diff
@@ -362,7 +361,6 @@ void Processor::run() {
         // Read radio reverse packets
         _radio->receive();
 
-        _loopMutex.lock();
```

Comment:
This mutex lock was ridiculous and I don't see any real reason for it other than the radiopacket.